### PR TITLE
Frediuk/2challenge

### PR DESCRIPTION
**Pull request by frediukos**

### CODE REVIEW

File: frediuk-olena/user/challenge-1.spec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Кто твое домашнее животное?
+
+Description
+
+Создайте переменную `pet` и присвойте ей значение 'Dog'.
+
+Приведем пример, который принципиально похож на то что нужно сделать,
+но адаптируйте его к требованиям задачи:
+```
+const pet = 'Golden Fish';
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-1.spec.js]: change spell to Gold Fish

File: frediuk-olena/user/challenge-2.spec.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+/*
+Name
+Можешь ли ты сегодня играть на iPad?
+
+Description
+
+У тебя очень строгие родители, которые установили правило:
+Ты можешь играть на планшете только если оценка по математике не ниже "В" и все домашние задания выполнены.
+
+Создайте переменную `mark` и присвойте ей значение false (оценка ниже "В").
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: change 'Mark' to 'Grade'

File: frediuk-olena/user/challenge-2.spec.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Создайте переменную `homeTask` и присвойте ей значение true (все домашние задания выполнены).
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: change homeTask to HomeWork

File: frediuk-olena/user/challenge-2.spec.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+Создайте переменную `iPad` присвойте ей условие проверки доступа к планшету.
+Запустите программу. Можно ли играть на iPad?
+
+Переприсвойте значения переменных `mark` и `homeTask` на такие,
+которые позволят играть на планшете.
+
+*/
+
+// Solution
+const mark = true;
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: replace mark to grade

File: frediuk-olena/user/challenge-2.spec.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+const homeTask = true;
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: replace homeTask to homeWork

File: frediuk-olena/user/challenge-2.spec.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+const iPad = mark && homeTask;
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: replace mark and hometask

File: frediuk-olena/user/challenge-2.spec.js [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+
+// Tests
+
+it('Переменная `mark` существует и имеет значение', () => {
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: grade

File: frediuk-olena/user/challenge-2.spec.js [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+    expect(mark).not.undefined;
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: grade

File: frediuk-olena/user/challenge-2.spec.js [hunk 9]
```diff
@@ -0,0 +1,56 @@
+const { expect } = require('chai');
+
+/*
+Name
+Можешь ли ты сегодня играть на iPad?
+
+Description
+
+У тебя очень строгие родители, которые установили правило:
+Ты можешь играть на планшете только если оценка по математике не ниже "В" и все домашние задания выполнены.
```
Comment (KatAvdeeva) [inline frediuk-olena/user/challenge-2.spec.js]: на планшете, только если

File: frediuk-olena/user/challenge-2.spec.js [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+const iPad = mark && homeTask;
+
+
+// Tests
+
+it('Переменная `mark` существует и имеет значение', () => {
+    expect(mark).not.undefined;
+});
+
+it('Переменная `homeTask` существует и имеет значение', () => {
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: homework

File: frediuk-olena/user/challenge-2.spec.js [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+    expect(homeTask).not.undefined;
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: homework

File: frediuk-olena/user/challenge-2.spec.js [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+});
+
+it('Переменная `mark` содержит тип данных boolean', () => {
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: grade

File: frediuk-olena/user/challenge-2.spec.js [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+    expect(mark).a('boolean');
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: grade

File: frediuk-olena/user/challenge-2.spec.js [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+});
+
+it('Переменная `homeTask` содержит тип данных boolean', () => {
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: homework

File: frediuk-olena/user/challenge-2.spec.js [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+    expect(homeTask).a('boolean');
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: homework

File: frediuk-olena/user/challenge-2.spec.js [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+});
+
+it('Переменная `mark` содержит значение  true', () => {
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: grade

File: frediuk-olena/user/challenge-2.spec.js [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+    expect(mark).eq(true);
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: grade

File: frediuk-olena/user/challenge-2.spec.js [hunk 18]
```diff
@@ -0,0 +1,56 @@
...
+});
+
+it('Переменная `homeTask` содержит значение  true', () => {
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: homework

File: frediuk-olena/user/challenge-2.spec.js [hunk 19]
```diff
@@ -0,0 +1,56 @@
...
+    expect(homeTask).eq(true);
```
Comment (nsilchuk) [inline frediuk-olena/user/challenge-2.spec.js]: homework

**Pull request closed without merging**